docs/sources/alerting/manage-notifications/template-notifications/write-templates.md

```diff
@@ -0,0 +1,252 @@
+---
+aliases:
+keywords:
+  - grafana
+  - alerting
+  - notifications
+  - templates
+  - write templates
+title: Write templates
+weight: 400
+---
+
+# Write templates
+
+You write message templates in Go's templating language, [text/template](https://pkg.go.dev/text/template). In the following topics we will look at how to write message templates, and then look at a number of more complex examples that you might want to use as a reference for customizing your notifications.
```

Comment:
```suggestion
You write message templates in Go's templating language, [text/template](https://pkg.go.dev/text/template). In the following topics, we look at how to write notification templates and provide more complex examples for you to use as a reference for customizing your notification messages.
```

Reply:
Can we have a step by step here on how a user writes templates? Is there any order a user needs to follow?

Reply:
It's difficult to tell the user exactly what to write because with customization the user gets to make the template look how they want. There is a narrative on this page that guides the user on how to structure their template which might be sufficient?